test/v2_integration.py

```diff
@@ -1299,11 +1299,7 @@ def test_ocsp():
     # checking OCSP until we either see a good response or we timeout (5s).
     verify_ocsp(cert_file.name, "/hierarchy/intermediate-cert-rsa-a.pem", "http://localhost:4002", "good")
 
-# TODO(#5938): Remove _operator suffix from this test and remove CONFIG_NEXT check.
 def test_ct_submission_operator():
```

Comment:
Can remove the _operator() suffix here, since the `getOperatorSCTs` function doesn't exist anymore.